sound/soc/sof/intel/hda.c

```diff
@@ -414,6 +415,22 @@ void hda_dsp_dump_skl(struct snd_sof_dev *sdev, u32 flags)
 	}
 }
 
+static void hda_dsp_dump_ext_rom_status(struct snd_sof_dev *sdev)
+{
+	u32 value;
+	int i;
+
+	dev_err(sdev->dev, "error: extended rom status:");
+
+	for (i = 0; i < HDA_EXT_ROM_STATUS_SIZE; i++) {
+		value = snd_sof_dsp_read(sdev, HDA_DSP_BAR, HDA_DSP_SRAM_REG_ROM_STATUS + i * 0x4);
+		if (i == (HDA_EXT_ROM_STATUS_SIZE - 1))
+			dev_err(sdev->dev, "0x%x ", value);
+		else
+			dev_err(sdev->dev, "0x%x\n", value);
```

Comment:
How useful are these 8*4 bytes? How does this help us without a description of what those bytes are?

Reply:
@plbossart  apparently it's useful according to @mmaka1. He was the one who suggested adding these. As for the details of what they  represent, in his own words " must remain obfuscated and represented only as extended ROM status"

Reply:
ok, then I think we should clarify this for upstream that this information is added to help with remote support, but requires access to non-public information to be useful.

Reply:
done!